packages/server-wallet/src/wallet/index.ts

```diff
@@ -187,7 +187,7 @@ export class Wallet implements WalletInterface {
             channelIds.push(channelId);
           } else {
             ss.signatures?.map(sig => channel.addState(ss, sig));
-            await Channel.query(tx).update(channel);
+            await Channel.query(tx).upsertGraph(channel);
```

Comment:
I don't understand the context of this change, since it's made in a single, large commit. Could you comment on why it's necessary?

Reply:
I think Alex identified it in #2370 but `update` causes a duplicate channel_id error in Postgres